src/view/gui/fMainWindow.h

```diff
@@ -1476,6 +1481,7 @@ public slots:
   vtkSmartPointer<vtkImageData> mMask;
 
   SHAPE_MODE m_drawShapeMode;
+  int mSelectedMaskOpacity;
```

Comment:
@AlexanderGetka-cbica This shouldn't be in mainwindow. Did you check fDrawingPanel::getCurrentOpacity()? 
You may not need an ivar. If it's needed, it could be in drawingpanel or imagespanel.